homeassistant/components/velux/light.py

```diff
@@ -35,6 +35,7 @@ class VeluxLight(VeluxEntity, LightEntity):
 
     _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
     _attr_color_mode = ColorMode.BRIGHTNESS
+    _attr_name = None
```

Comment:
Well, is the light a main feature of a window? I would argue that the cover is the main feature of the window

Reply:
There are devices like this which are basically just a dimmer or simple on/off switch: https://shop.somfy.de/licht-empfanger-dimmbar-io.html (German, sorry). And these are supported by the Velux gateway, represented by https://github.com/Julius2342/pyvlx/blob/master/pyvlx/lightening_device.py. So there is no window (or blind, etc.) involved at all, just a light that you control via the Velux gateway.